R/util.R

```diff
@@ -159,6 +159,28 @@ hash_object <- function(object) {
   digest::digest(object)
 }
 
+file_size <- function(filenames, named = TRUE) {
```

Comment:
This can come out, but `file.size` will do this.  There's no point adding the error condition here though as this should be tested elsewhere (and it is I think) - the message as it stands would tie a generally named function to a specific task.  If a region of code is untestable it should not be here